collect_app/src/main/res/drawable/ic_room_blue_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24"
         android:viewportHeight="24">
     <path
-        android:fillColor="@color/blue_500"
+        android:fillColor="#2196F3"
```

Comment:
Why this color (and some other below) is not declared in colors.xml?

Reply:
I'll make a specific file for these colors as well.